src/proofs/proofs.service.ts

```diff
@@ -17,13 +20,16 @@ export class ProofsService {
   async getBouncerProof(chain: Chain, address: string): Promise<MerkleProof> {
     const fileName = `badger-bouncer-${parseInt(chain.chainId)}.json`;
     const bouncerFile = await getObject(REWARD_DATA, fileName);
+    console.log('--------------------------------------------------');
```

Comment:
is this for test or purposeful? 
if test we can remove probably dont wanna log this each time i guess?

Reply:
gosh, my bad, will rm